auth/api/groups/http/requests.go

```diff
@@ -53,10 +53,12 @@ func (req updateGroupReq) validate() error {
 }
 
 type listGroupsReq struct {
-	token    string
-	id       string
-	level    uint64
-	tree     bool // If `true` result is JSON that represents groups hierarchy, otherwise JSON just holds array of groups.
+	token string
+	id    string
+	level uint64
+	tree  bool
+	// - `true`  - result is JSON tree representing groups hierarchy,
```

Comment:
I would comment next to the field. I prefer Object than tree. 
`true: the  result is  a JSON Object representing groups hierarchy.  / false:  the result is a JSON array representing groups.`

Reply:
object is generic and tree is exactly what it was

Reply:
ok